extensions/podman/src/extension.ts

```diff
@@ -35,6 +35,8 @@ type StatusHandler = (name: string, event: extensionApi.ProviderConnectionStatus
 
 const listeners = new Set<StatusHandler>();
 const podmanMachineSocketsDirectoryMac = path.resolve(os.homedir(), '.local/share/containers/podman/machine');
+const podmanMachineSocketsSymlinkDirectoryMac = path.resolve(os.homedir(), '.podman');
+const macosMaxSocketPathLength = 104
```

Comment:
hello, I would probably name it with uppercase like MACOS_MAX_SOCKET_PATH_LENGTH
WDYT ?